cg/meta/workflow/rnafusion.py

```diff
@@ -358,21 +363,38 @@ def get_multiqc_json_metrics(self, case_id: str) -> List[MetricsBase]:
             for metric_name, metric_value in metrics_values.items()
         ]
 
-    def write_metrics_deliverables(self, case_id: str, dry_run: bool = False):
+    @staticmethod
+    def ensure_mandatory_metrics_present(metrics: List[MetricsBase]) -> None:
+        """Check that all mandatory metrics are present. Raise MissingMetrics error if missing."""
```

Comment:
```suggestion
        """Check that all mandatory metrics are present. Raise error if missing."""
```
It might change in the future and it will be hard to remember to change in the doc str

Reply:
good point!